src/world_interface/world_interface.h

```diff
@@ -28,6 +30,9 @@ enum class WorldInterface { real, sim2d, sim3d, noop };
 /** @brief The current world interface being used. */
 extern const WorldInterface WORLD_INTERFACE;
 
+const DiffDriveKinematics driveKinematics();
+const DiffWristKinematics wristKinematics();
```

Comment:
Is there any point in returning a const object? The object's still being copied everywhere, isn't it?